src/common/POLFields/FieldHolding/useHoldings.js

```diff
@@ -0,0 +1,20 @@
+import { useQuery } from 'react-query';
+
+import { useOkapiKy } from '@folio/stripes/core';
+import { LIMIT_MAX } from '@folio/stripes-acq-components';
+
+export const useHoldings = (instanceId) => {
+  const ky = useOkapiKy();
+  const searchParams = { query: `instanceId==${instanceId}`, limit: LIMIT_MAX };
+
+  const { isLoading: isHoldingsLoading, data = {} } = useQuery(
```

Comment:
I'd suggest to return full results of the `useQuery` from this hook, not only  `isLoading` and `data`.
(also not renaming `isLoading`)
And pass potential options to `useQuery` too
```js
export const useHoldings = (instanceId, options) => {
``` 
or
```js
export const useHoldings = ({instanceId, ...options}) => {
``` 
then pass it to `useQuery`
```js
  const query = useQuery(/*...*/, options)
``` 
And return like this
```js
  return {
    holdings: query.data?.holdingsRecords ?? [],
    ...query,
  };
```

This could be cleaner and more consistent API for the the `useQuery` wrapper, which we basically have here

Reply:
agree with `export const useHoldings = (instanceId, options) => {`, but don't think we always need to return everything from query, we define return type interface

or we should define must have fields from react-query to always return

Reply:
Returning everything is useful because we don't need to bother remembering API - it's in the docs of react-query (plus our own specific addition).

Plus, this way our hook is more compliant to Open-Closed principle: it's easier to extend without modification.